### PR TITLE
Add note about visually hidden text in styled radio group

### PR DESCRIPTION
**Pull request by joe-watkins**
Adding notes to the SRG example

### CODE REVIEW

File: _checklist-web/radio.md [hunk 1]
```diff
@@ -169,6 +169,8 @@ This custom button requires extra scripting work for roving tabindex and event l
 - By default, it is not expected behavior that each radio button can be tabbed to. This is how radio buttons naturally behave
 - As soon as a radio button is selected, the selected radio input receives focus first from the group. As a result screen reader users may not discover a nested control for an option if they start switching between radio buttons alone
 - To try to mitigate screen reader users not discovering the nested controls, describe the fieldset / radiogroup with non-visual text. This can be done with <code>aria-describedby</code> on the <code>fieldset</code>. For example, "Edit controls are available which follow each radio button"
+  - <strong>Critical:</strong> Do not include this visually hidden text if the radio group does not consist of nested controls. It should not be present in the DOM. CSS <code>display: none;</code> is not a sufficient method for hiding it from screen readers. In addition to the text not being in the DOM if not needed, ensure that the <code>aria-describedby</code> on the <code>fieldset</code> does not reference an <code>ID</code> that is not in the DOM.
```
Comment (silverli) [inline _checklist-web/radio.md:172]: I don't know if this is needed @joe-watkins. I think the comment in the code will be the most helpful thing for devs to cross check

File: _includes/examples/input-radio-styled-group.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <fieldset class="checkbox-radio-group" aria-describedby="styled-radio-group-helper-text">
   <legend>Choose your payment method:</legend>
-  <!-- visually hidden helper text describing fieldset -->
+  <!-- Visually hidden helper text describing fieldset -->
+  <!-- This text should only be present in the DOM if the radio group has nested controls. Hiding it with CSS display: none; is not enough to hide it from screen readers. Ensure aria-describedby on the fieldset does not point to an ID that is not in the DOM -->
```
Comment (silverli) [inline _includes/examples/input-radio-styled-group.html:4]: This is great!

File: _checklist-web/radio.md (hunk 1 above)
Comment (silverli) [inline _checklist-web/radio.md:172]: I don't know if this is needed @joe-watkins. I think the comment in the code will be the most helpful thing for devs to cross check

**Review by silverli — CHANGES_REQUESTED** [review]
One quick thought for you

File: _checklist-web/radio.md (hunk 1 above)
Comment (joe-watkins) [reply _checklist-web/radio.md]: @silverli Good call - I've removed this bullet

**Pull request merged**